metadb/database/service.go

```diff
@@ -120,18 +119,16 @@ func (db *Ops) ReadUserData(ctx context.Context, artistPastelID string) (userdat
 
 // Run run the rqlite database service
 func (db *Ops) Run(ctx context.Context) error {
+	var err error
 
 	ctx = log.ContextWithPrefix(ctx, logPrefix)
 	log.WithContext(ctx).Info("start initialization")
 
-	content, err := ioutil.ReadFile(db.config.SchemaPath)
```

Comment:
Please remove this config var from config struct as well

Reply:
@mateeullahmalik : let keep it. later maybe we will have bunch of template file.
we're thinking to support both of them - that should be easy for development later.